src/components/structures/MatrixChat.tsx

```diff
@@ -1132,6 +1133,29 @@ export default class MatrixChat extends React.PureComponent<IProps, IState> {
                 ));
             }
         }
+
+        const client = MatrixClientPeg.get();
+        const plEvent = roomToLeave.currentState.getStateEvents(EventType.RoomPowerLevels, '');
+        const plContent = plEvent ? plEvent.getContent() : {};
+        const userLevels = plContent.users || {};
+        const currentUserLevel = userLevels[client.getUserId()];
+        const userLevelValues = Object.values(userLevels);
+        if (isNumberArray(userLevelValues)) {
+            const maxUserLevel = Math.max(...userLevelValues);
+            // If the user is the only user with highest power level
+            if (maxUserLevel === currentUserLevel &&
+                userLevelValues.lastIndexOf(maxUserLevel) == userLevelValues.indexOf(maxUserLevel)) {
+                warnings.push((
+                    <span className="warning" key="last_admin_warning">
+                        { ' '/* Whitespace, otherwise the sentences get smashed together */ }
+                        { _t("You are the sole person with the highest role in this room. " +
+                        "If you leave, the room could become unmoderable. Consider giving " +
```

Comment:
`unmoderable` doesn't feel like a word. `unmoderatable` sounds better to me but I also don't see it as a real word so probably need to rearrange the prose to accommodate proper English. Perhaps:

"If you leave, the room might not be able to be moderated anymore."